src/model/enums/country.rs

```diff
@@ -1,11 +1,11 @@
 use serde::{Deserialize, Serialize};
-use strum::ToString;
+use strum::{AsRefStr, ToString};
 
 /// ISO 3166-1 alpha-2 country code, from
 /// [country-list](https://datahub.io/core/country-list)
 ///
 /// [Reference](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
-#[derive(Clone, Serialize, Deserialize, Copy, PartialEq, Eq, Debug, ToString)]
+#[derive(Clone, Serialize, Deserialize, Copy, PartialEq, Eq, Debug, ToString, AsRefStr)]
```

Comment:
If we're using `AsRefStr` we might as well remove the `ToString`, right? (this happens in many other places)

Reply:
Removed :)